samples/matter/manufacturer_specific/README.rst

```diff
@@ -261,9 +262,16 @@ To test ``NordicDevkit`` cluster's attributes and commands, complete the followi
    .. parsed-literal::
       :class: highlight
 
-      any subscribe-by-id 0xFFF1FC01 3 0 120 1 1
+      any subscribe-by-id 0xFFF1FC01 0xFFF10002 0 120 1 1
 
 #. Press the button assigned to the ``UserButton`` and check if the attribute state is updated in the chip-tool.
+#. Read the ``UserButtonChanged`` event to check that events were generated on ``UserButton`` attribute changes.
```

Comment:
Nit: Can we show users how to subscribe to this event?

Reply:
Discussed f2f.